practice-one/src/components/food-card.html

```diff
@@ -0,0 +1,14 @@
+<div class="card-container">
+  <div class="card-filter">
+      <div class="food-image"><img src="./assets/images/kebab.png" alt=""></div>
+      <div class="food-price"><p>10$</p></div>
+  </div>
+  <div class="food-rating"><img src="./assets/images/kebab-rating.png" alt=""></div>
+  <p class="food-card-header">Kebab</p>
+  <p class="food-card-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Sit repellendus repudiandae quod error? Distinctio provident enim odio sunt, ea, ullam eligendi dolore non saep.</p>
```

Comment:
This is not correct, we can pass price, name of the food, description of the disk like we pass `text` of `button.html`

Reply:
i fixed it